code/modules/chemistry/tools/patches.dm

```diff
@@ -477,7 +477,7 @@ TYPEINFO(/obj/item/reagent_containers/mender)
 
 	on_reagent_change(add)
 		..()
-		if (src.reagents)
+		if (src.reagents && src.reagents.total_temperature > 330)
```

Comment:
?

Reply:
infinite loop triggered by adding reagents_changed callbacks to temperature_reagents. 

makes more sense that heating reagents also counts as changing them, better place to hook it.

Reply:
Should also probably check if temp is lower than 270 though.